packages/astro/src/core/dev/container.ts

```diff
@@ -156,7 +158,7 @@ export async function runInContainer(
 	params: CreateContainerParams,
 	callback: (container: Container) => Promise<void> | void
 ) {
-	const container = await createContainer({ ...params, disableTelemetry: true });
+	const container = await createContainer({...{ mode: 'developmen' }, ...params, disableTelemetry: true });
```

Comment:
![image](https://user-images.githubusercontent.com/27261522/229667371-58db8fef-6105-4f84-9e79-59dddd0dcb11.png)
I did what you said. but it will give me a ts error.